app/controllers/pages_controller.rb

```diff
@@ -4,5 +4,18 @@ def home
   end
 
   def about
+    # TODO
   end
+
+  def search
+    if params[:search].blank?
+      flash[:alert] = "Search cannot be blank"
+      redirect_to root_path
+    else
+      parameter = params[:search].downcase
+      @articles = Article.where('lower(title) LIKE :search OR lower(description) LIKE :search', 
+                  search: "%#{parameter}%").order("title ASC")
```

Comment:
Super! Zur besseren Lesbarkeit würde ich es vermutlich eher so schreiben:

```suggestion
      @articles = Article
        .where('lower(title) LIKE :search OR lower(description) LIKE :search', search: "%#{parameter}%")
        .order("title ASC")
```